### ISSUE TITLE
Release 4.6.0 - Release Candidate 2 - Workload benchmarks metrics

### ISSUE THREAD

**Issue by fdalmaup** [wazuh/wazuh#19845]
The following issue aims to run all `workload benchmarks` for the current release candidate, report the results, and open new issues for any encountered errors.

## Workload benchmarks metrics information
|                                               |                                            |
|-----------------------------------------------|--------------------------------------------|
| **Main release candidate issue**              |                 #19833                           |
| **Version**                                   |                     4.6.0                       |
| **Release candidate #**                       |                 RC 2                           |
| **Tag**                                       |                 [v4.6.0-rc2](https://github.com/wazuh/wazuh/tree/v4.6.0-rc2)                           |
| **Previous Workload benchmarks metrics issue**|                        #19523                    |

## Test configuration
All tests will be run and workload performance metrics will be obtained for the following clustered environment configurations:

|                   |                    |
|-------------------|--------------------|
| **# Agents**      | **# Worker nodes** |
|         50000          |      25              |

## Test report procedure
All individual test checks must be marked as:
|                                  |                                            |
|---------------------------------|--------------------------------------------|
| Pass | The test ran successfully. |
| Xfail | The test was expected to fail and it failed. It must be properly justified and reported in an issue.  |
| Skip | The test was not run. It must be properly justified and reported in an issue.  |
| Fail | The test failed. A new issue must be opened to evaluate and address the problem. |

All test results must have one the following statuses: 
|                                  |                                            |
|---------------------------------|--------------------------------------------|
| :green_circle:  | All checks passed. |
| :red_circle:  | There is at least one failed check. |
| :yellow_circle:  | There is at least one expected fail or skipped test and no failures. |

Any failing test must be properly addressed with a new issue, detailing the error and the possible cause. It must be included in the `Fixes` section of the current release candidate main issue.

Any expected fail or skipped test must have an issue justifying the reason. All auditors must validate the justification for an expected fail or skipped test.

An extended report of the test results must be attached as a zip or txt. This report can be used by the auditors to dig deeper into any possible failures and details.

## Conclusions

All tests have been executed and the results can be found [here](https://github.com/wazuh/wazuh/issues/19845#issuecomment-1777891309).

All tests have passed and the fails have been reported or justified. I therefore conclude that this issue is finished and OK for this release candidate.

For a detailed report on the cluster performance metrics please refer to [this issue comment](https://github.com/wazuh/wazuh/issues/19845#issuecomment-1777935181).

## Auditors validation
The definition of done for this one is the validation of the conclusions and the test results from all auditors.

All checks from below must be accepted in order to close this issue.

- [x] @Selutario 
- [ ] @davidjiglesias

**Comment by fdalmaup** [comment]
# API Performance Tests

## Tests report

[API_Performance_Test_Report.zip](https://github.com/wazuh/wazuh/files/13126424/API_Performance_Test_Report.zip)

## Tests results
---
|                                  |                                            |                                           |            
|--|--|--|
| **Endpoint test name**  | **Status** | **Issues Ref.** |
| GET /cluster/local/info | :green_circle: | |
| GET /cluster/nodes| :green_circle:  | |
| GET /cluster/healthcheck | :green_circle: | |
| GET /cluster/status | :green_circle: | |
| GET /cluster/local/config | :green_circle: | |
| GET /cluster/api/config | :green_circle: | |
| GET /cluster/configuration/validation | :green_circle: | |
| GET /manager/status | :green_circle: | |
| GET /manager/info | :green_circle: | |
| GET /manager/configuration | :green_circle: | |
| GET /manager/logs | :green_circle:   | |
| GET /manager/api/config | :green_circle: | |
| GET /manager/configuration/validation | :green_circle: | |
| GET /mitre/groups | :green_circle: | |
| GET /mitre/metadata | :green_circle: | |
| GET /mitre/mitigations | :green_circle: | |
| GET /mitre/references | :green_circle: | |
| GET /mitre/software | :green_circle: | |
| GET /mitre/tactics | :green_circle: | |
| GET /mitre/techniques | :green_circle: | |
| GET /overview/agents | :green_circle:  | |
| GET /tasks/status | :green_circle: | |
| PUT /active-response | :yellow_circle: | https://github.com/wazuh/wazuh-qa/issues/1266 |
| PUT /rootcheck | :green_circle: | |
| PUT /syscheck | :green_circle: | |
| POST /groups | :green_circle: | |
| POST /security/users | :green_circle: | |
| POST /security/roles | :green_circle: | |
| POST /security/policies | :green_circle: | |
| POST /security/rules | :green_circle: | |
| PUT /agents/group | :yellow_circle: | Endpoint is not being tested (https://github.com/wazuh/wazuh-qa/issues/3665). This related issue was opened in the past: https://github.com/wazuh/wazuh/issues/13872 |
| PUT /agents/group/new_test_group/restart | :green_circle: | |
| PUT /agents/restart | :green_circle: |  |
| POST /agents | :green_circle: | |
| POST /agents/insert​|  :green_circle: | |
| POST /agents/insert​/quick | :green_circle: | |
| GET /agents | :green_circle: | |
| GET /agents/no_group | :green_circle: | |
| GET /agents/outdated | :green_circle: | |
| GET /decoders | :green_circle: | |
| GET /groups | :green_circle: | |
| GET /lists | :green_circle: | |
| GET /rules | :green_circle: | |
| GET /security/users | :green_circle: | |
| GET /security/roles | :green_circle: | |
| GET /security/policies | :green_circle: | |
| GET /security/rules | :green_circle: | |
| DELETE /groups | :green_circle: | |
| DELETE /agents | :green_circle: | |
| DELETE /security/users | :green_circle: | |
| DELETE /security/roles | :green_circle: | |
| DELETE /security/policies | :green_circle: | |
| DELETE /security/rules | :green_circle: | |
| PUT /manager/restart | :green_circle: | |
| PUT /cluster/restart | :green_circle: | |

**Comment by fdalmaup** [comment]
# Cluster

[workloadArtifacts.zip](https://github.com/wazuh/wazuh/files/13126739/workloadArtifacts.zip)

Using a modified version of the script given [here](https://github.com/wazuh/wazuh/issues/19523#issuecomment-1753424259), the errors found in the `cluster.log` files of the master and worker nodes were: 

<details><summary>Worker errors</summary>

    Searching for errors in artifacts/master/logs/cluster.log:
    -------------------------------------------
    Searching for errors in artifacts/worker_10/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:34 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_11/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_12/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_13/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:34 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_14/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_15/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:34 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_16/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_17/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:34 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_18/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_19/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:34 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_1/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_20/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_21/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_22/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_23/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_24/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_25/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_2/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_3/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:33 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_4/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_5/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:34 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_6/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_7/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_8/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------
    Searching for errors in artifacts/worker_9/logs/cluster.log:
    2023/10/24 17:43:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:20 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:30 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:40 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:43:50 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:00 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    2023/10/24 17:44:10 ERROR: [Local Server] [Main] Could not connect to master. Trying again in 10 seconds.
    -------------------------------------------



</details>
Which are due to expected workers' disconnections during the test. 
Comparing with the [last RC test](https://github.com/wazuh/wazuh/issues/19523), no `Error sending sendsync response to local client: Error 3020 - Timeout sending request: ok` message was found since the fix carried out in #19629.

No errors were found in the master logs.

## Cluster tasks duration

```json
{
    "setup_phase": {
        "agent-info_sync": {
            "time_spent(s)": {
                "workers": {
                    "mean": ("worker_21", 0.177575221238938),
                    "max": ("worker_6", 1.416)
                },
                "master": {
                    "mean": ("master", 0.07790500927643784),
                    "max": ("master", 1.25)
                }
            }
        },
        "integrity_check": {
            "time_spent(s)": {
                "workers": {
                    "mean": ("worker_18", 0.1022047244094488),
                    "max": ("worker_13", 1.394)
                },
                "master": {
                    "mean": ("master", 0.00567161351436691),
                    "max": ("master", 0.082)
                }
            }
        },
        "integrity_sync": {
            "time_spent(s)": {
                "workers": {
                    "mean": ("worker_14", 0.03815909090909091),
                    "max": ("worker_24", 0.093)
                },
                "master": {
                    "mean": ("master", 0.23994589552238804),
                    "max": ("master", 1.466)
                }
            }
        }
    },
    "stable_phase": {
        "agent-info_sync": {
            "time_spent(s)": {
                "workers": {
                    "mean": ("worker_10", 0.013481481481481483),
                    "max": ("worker_10", 0.338)
                },
                "master": {
                    "mean": ("master", 0.14216666666666666),
                    "max": ("master", 0.175)
                }
            }
        },
        "integrity_check": {
            "time_spent(s)": {
                "workers": {
                    "mean": ("worker_5", 0.009758620689655174),
                    "max": ("worker_5", 0.034)
                },
                "master": {
                    "mean": ("master", 0.0037731673582295996),
                    "max": ("master", 0.023)
                }
            }
        }
    }
}
```

**Comment by damarisg** [comment]
LGTM!
